data/templates/ipsec/ike-esp.tmpl

```diff
@@ -0,0 +1,32 @@
+{% macro conn(ike, ike_ciphers, esp, esp_ciphers) -%}
+{%  if ike %}
+{%      if "key_exchange" in ike %}
+    keyexchange = {{ ike.key_exchange }}
+{%      endif %}
+    ike = {{ ike_ciphers }}
+{%      if "lifetime" in ike %}
+    ikelifetime = {{ ike.lifetime }}s
+{%      endif %}
+    reauth = {{ ike.ikev2_reauth if "ikev2_reauth" in ike else "no" }}
+    closeaction = {{ ike.close_action if "close_action" in ike else "none" }}
+{%      if "dead_peer_detection" in ike %}
+    dpdaction = {{ ike.dead_peer_detection.action }}
+    dpdtimeout = {{ ike.dead_peer_detection.timeout }}
+    dpdinterval = {{ ike.dead_peer_detection.interval }}
```

Comment:
Is this parameter correct?
```
 set vpn ipsec ike-group grp-IKE dead-peer-detection interval 32

vyos@r4-1.3:~$ sudo ipsec restart
Stopping strongSwan IPsec...
Starting strongSwan 5.7.2 IPsec [starter]...
# unknown keyword 'dpdinterval'
# unknown keyword 'dpdinterval'
# unknown keyword 'dpdinterval'
### 3 parsing errors (0 fatal) ###
vyos@r4-1.3:~$ 
vyos@r4-1.3:~$ sudo cat /etc/ipsec.conf | grep dpdi
    dpdinterval = 32

```
Most likely it is "delay"
https://wiki.strongswan.org/projects/strongswan/wiki/ConnSection

Reply:
Fixed the typo.